### ISSUE TITLE
Mapping assist options

### ISSUE THREAD

**Issue by ghost** [Hatchie-Studio/lotstracker#19]
In order to facilitate user scripted mapping the addition of a 'showroomid' or similar setting for mortals would be very useful.

**Comment by XanderAshe** [comment]
Useful for mapping yes. Bad for zones that are mazes

**Comment by ghost** [comment]
Improving functionality shouldn't be shot down by the mild inconvenience of slightly rebuilding some maze exits.  That's all it would take to keep mazes relevant if that is the turn off.  Make the maze harder.

Or implement an out of band protocol to transmit roomid that will never be seen but can still be used for mapping.  One can then add a builder room creation variable that enables or disables this out of band send for that specific room.  (a noidsend flag, or similar) Obviously this is more complex but you get to keep your precious mazes they way they are.  

Can we talk about how mazes are kinda ridiculous considering everyone is carrying around a datapad?

**Comment by XanderAshe** [comment]
Fair enough. Just a thought really. One would assume a datapad could help
one navigate an unfamiliar environment if the protocols were met.   However
it wouldn't be hard to download most maps for a fee for example.
